src/App.js

```diff
@@ -1,27 +1,35 @@
-import logo from './logo.svg';
+import { Toaster } from 'react-hot-toast';
+import { Route, Routes } from 'react-router-dom';
+import React from 'react';
+import Login from './pages/auth/login';
+import Register from './pages/auth/register';
+import CarsList from './pages/Home';
+import CarDetails from './components/EachCar';
+import NewCar from './pages/NewCar';
+import DeleteCar from './pages/DeleteCar';
+import Navbar from './components/Navigation/Navbar';
+import FixedNavbar from './components/Navigation/FixedNavbar';
 import './App.css';
+import './index.css';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://github.com/Bludidi/vehi-go-frontend/assets/57421823/c282f8e7-0678-471b-b90b-baf2e0609428)



That way, your project will be well documented. 💪